hbase-shell/src/main/ruby/shell/commands/assign.rb

```diff
@@ -22,8 +22,8 @@ module Commands
     class Assign < Command
       def help
         <<-EOF
-Assign a region. Use with caution. If region already assigned,
-this command will do a force reassign. For experts only.
+Assign a region. It could be executed only when region in expected state(CLOSED, OFFLINE).
```

Comment:
Can we mention hbck2 alternative commands?

Reply:
Yeah, will fix later. Thanks.

Reply:
Is this right place to provide doc reference links? If so, we can provide link to HBCK2 command doc: https://github.com/apache/hbase-operator-tools/blob/master/hbase-hbck2/README.md

Reply:
Good idea, I think so. Will fix later. Thanks.

Reply:
Nit: Instead of `you can use "assigns" which supported by HBCK2`, let's say `you can use "assigns" command available on HBCK2 tool`